geostore/check.py

```diff
@@ -13,3 +13,5 @@ class Check(Enum):
     STAGING_ACCESS = "staging bucket access"
     UNKNOWN_CLIENT_ERROR = "unknown client error"
     UNKNOWN_MULTIHASH_ERROR = "unknown multihash error"
+    UPLOADED_ASSETS_SHOULD_BE_CATALOG_OR_COLLECTION_ERROR = "assets should be catalog or collection"
```

Comment:
An *asset* should never be a catalogue or collection - an asset is a non-metadata file like a JPEG or SVG. How about something like `INVALID_STAC_ROOT_TYPE`? The error message in `geostore/check_stac_metadata/utils.py` already expands on what that means.

The same issue applies to `InvalidAssetFileError`.

Reply:
Couldn't agree more. Updated.